target/scripts/check-for-changes.sh

```diff
@@ -5,48 +5,40 @@
 
 # shellcheck source=./helpers/index.sh
 source /usr/local/bin/helpers/index.sh
+# determine `POSTMASTER_ADDRESS` for alias (re-)generation
+# shellcheck source=/dev/null
+source /etc/dms-settings
 
-function _log_date
-{
-  date +"%Y-%m-%d %H:%M:%S"
-}
+_log 'debug' "$(_log_date) Starting changedetector"
 
-LOG_DATE=$(_log_date)
-_notify 'task' "${LOG_DATE} Start check-for-changes script."
-
-# ? --------------------------------------------- Checks
-
-cd /tmp/docker-mailserver || exit 1
+if ! cd /tmp/docker-mailserver &>/dev/null
+then
+  _log "Could not change into '/tmp/docker-mailserver/' directory"
+  exit 1
+fi
 
```

Comment:
Likewise, would `_exit_with_error` be a better choice here?

Reply:
👍🏼